sdk/core/azure-core-serializer-json-jackson/src/main/java/com/azure/core/serializer/json/jackson/JacksonAdapter.java

```diff
@@ -123,13 +129,13 @@ public byte[] serializeToBytes(Object object, SerializerEncoding encoding) throw
 
         if (encoding == SerializerEncoding.XML) {
             return supportsXmlSerializable(object.getClass())
-                ? ReflectionSerializable.serializeXmlSerializableToBytes(object)
+                ? serializeXmlSerializableToBytes((XmlSerializable<?>) object)
```

Comment:
Ah - we have both xml and json support in json-jackson package. Package name is misleading.